server/mediaserver.go

```diff
@@ -386,6 +404,25 @@ func jsonProfileToVideoProfile(resp *authWebhookResponse) ([]ffmpeg.VideoProfile
 	return profiles, nil
 }
 
+func jsonDetectionToDetectorProfile(resp *authWebhookResponse) core.DetectionConfig {
```

Comment:
```suggestion
func jsonDetectionToDetectionConfig(resp *authWebhookResponse) core.DetectionConfig {
```

^ Feels a bit more accurate naming wise given that the return value is `core.DetectionConfig`. WDYT?